src/zjs_ipm.h

```diff
@@ -95,6 +124,15 @@ typedef struct zjs_ipm_message {
             uint8_t color_b;
             void *buffer;
         } glcd;
+#endif
+
+#ifdef BUILD_MODULE_SENSOR
+        struct sensor_data {
+            enum sensor_channel channel;
+            uint32_t frequency;
+            double value[3];
```

Comment:
ok yeah, this is the internal implementation, and only specific to Arduino 101 where we have to channel the data, so we can fix it up however we want, but that's a good idea.